README.md

```diff
@@ -165,32 +165,17 @@ This call is non-blocking and there is no response to check.
 If you want to record large numbers of the same event at once, or perhaps measure usage in terms of a unit like tokens or memory, you can optionally specify a quantity for your event:
 
 ```go
-import (
-  "context"
-  "os"
-
-  option "github.com/schematichq/schematic-go/option"
-  schematicclient "github.com/schematichq/schematic-go/client"
-  schematicgo "github.com/schematichq/schematic-go"
-)
-
-func main() {
-  apiKey := os.Getenv("SCHEMATIC_API_KEY")
-  client := schematicclient.NewSchematicClient(option.WithAPIKey(apiKey))
-  defer client.Close()
-
-  client.Track(context.Background(), &schematicgo.EventBodyTrack{
-    Event: "query-tokens",
-    Company: map[string]stringh{
-      "id": "your-company-id",
-    },
-    User: map[string]string{
-      "email":   "wcoyote@acme.net",
-      "user-id": "your-user-id",
-    },
-    Quantity: schematicgo.Int(1500),
-  })
-}
+client.Track(context.Background(), &schematicgo.EventBodyTrack{
```

Comment:
Because the imports for this library are annoying (i.e. you may have to import from several different packages), I prefer having the imports present in code snippets in the readme. Otherwise, if you're just here for this snippet, it might not be clear what "schematicgo" is